plugins/modules/zos_backup_restore.py

```diff
@@ -187,6 +187,7 @@
     description:
       - Specifies the new HLQ to use for the data sets being restored.
       - Defaults to running user's username.
+      - Use keywords R or K to restores datasets to their original HLQ from when they were archived.
```

Comment:
`restores` should be `restore`

- Use keywords R or K to restore datasets to their original...